Mathlib/Analysis/NormedSpace/Dual.lean

```diff
@@ -53,35 +53,15 @@ variable (E : Type*) [SeminormedAddCommGroup E] [NormedSpace 𝕜 E]
 variable (F : Type*) [NormedAddCommGroup F] [NormedSpace 𝕜 F]
 
 /-- The topological dual of a seminormed space `E`. -/
-def Dual :=
+abbrev Dual :=
   E →L[𝕜] 𝕜
```

Comment:
```suggestion
abbrev Dual : Type _ := E →L[𝕜] 𝕜
```

Reply:
It's a stylistic change for the reader: lean doesn't care.

It's meant to make it clear that we're defining a type; but more generally, the convention in mathlib is to pretty much always write the full type to help the reader. For example, we prefer to write `def choose (m n : ℕ) : ℕ := sorry` instead of `def choose (m n : ℕ) := sorry`. This is just another case of the same thing.

Reply:
I added the signature.